docs/serverless/ingest/auto-import.mdx

```diff
@@ -0,0 +1,89 @@
+---
+slug: /serverless/security/auto-import
+title: Automatic import
+description: Use automatic import to quickly ingest third-party data.
+tags: [ 'serverless', 'security', 'how-to' ]
+status: in review
+---
+
+<DocBadge template="technical preview" />
+
+<DocCallOut title="Technical preview" color="warning">
+This feature is in technical preview. It may change in the future, and you should exercise caution when using it in production environments. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of GA features.
+</DocCallOut>
+
+This page explains how to use Automatic Import. Automatic Import helps you quickly parse, ingest, and create [ECS mappings](https://www.elastic.co/elasticsearch/common-schema) for data from sources that don't yet have prebuilt Elastic integrations. This can accelerate your migration to Elastic's SIEM, and help you quickly add new data sources to an existing SIEM deployment. Automatic Import uses a large language model (LLM) with specialized instructions to quickly analyze your source data and create a custom integration. 
```

Comment:
I'm flexible but I think the "quickly" is important here because this feature doesn't enable a completely new functionality - people could already create their own custom integrations - but it could take a lot of labor. This makes it much easier/faster. What do other reviewers think?

Reply:
I think we could omit this instance as @jmikell821 suggests, though I also agree with @benironside that in general, it's appropriate since Auto Import _does_ actually save time and effort. "Quickly" already shows up at the beginning of the paragraph and in the page description, so that establishes the idea pretty well.

On `SIEM`, our docs historically haven't used it much but I don't think that means we should avoid it — we know that Elastic is renewing its focus on SIEM as a core customer use case. I think the term itself works better as an adjective/descriptor ("a SIEM use case" or "a SIEM solution") than a noun, since the UI doesn't have anything called "the Elastic SIEM" or "Elastic's SIEM." Something like this:

```suggestion
Automatic Import helps you quickly parse, ingest, and create [ECS mappings](https://www.elastic.co/elasticsearch/common-schema) for data from sources that don't yet have prebuilt Elastic integrations. This can accelerate your migration to ((elastic-sec)), and help you quickly add new data sources to an existing SIEM solution in ((elastic-sec)). Automatic Import uses a large language model (LLM) with specialized instructions to quickly analyze your source data and create a custom integration. 
```

☝️ This also omits the first sentence, which seems unnecessary — I'd be surprised if a page called "Automatic Import" _didn't_ explain how to use Automatic Import.